subroutines.sh

```diff
@@ -3,39 +3,71 @@
 # subroutines.sh
 #
 
+
 # Function
 send_command() {
   vin=$1
-  shift
+  cmd=$2
+  cmd_name=$3
+
+  TESLA_CONTROL_CMD='/usr/bin/tesla-control -ble -vin $vin -key-name /share/tesla_blemqtt/${vin}_private.pem -key-file /share/tesla_ble_mqtt/${vin}_private.pem $cmd 2>&1'
```

Comment:
Shouldn't we use braces around cmd? ${cmd}? It should work without though

Reply:
Unless I have something like `${var:14}`  or `filename${vin}` or `file${vin}name`  ... if the variable has a space before and after, usually I don't add the  { } then ...  in the following case, `FILENAMES$vin` , visually I find it's easier to spot the var w/ {} `FILENAMES${vin}`